### ISSUE TITLE
Using Cinder / OpenGL functions with "native" OpenGL context

### ISSUE THREAD

**Issue by KerstinKeller** [cinder/Cinder#1924]
I've been experiencing with Cinder a bit and like the framework and particularly the OpenGL abstraction.

Now I am wondering if it is possible to use the OpenGL functionality of Cinder without inheriting from `ci::app::App` and using the `CINDER_APP( BasicApp, RendererGl )` macro.

The background is that we have our own GUI framework with logging / different tabs / a native OpenGL context and I would like to use the additional functionality from our GUI combined with the cinder OpenGL abstraction.
Just calling / using the cinder functions unfortunately leads to crashes. 

Does anybody have any thoughts on this topic?

**Comment by richardeakin** [comment]
Yes, totally possible, you'll have to make your own `RendererGl` the same way that that macro does, though. It isn't the usual route so could possibly be made better, and you have to hand the platform-specific `RendererGl` some things like an `HWND` on windows for example, but it shouldn't require for an `App` to exist.
